SampleApps/WebView2_WinUI3_Sample/readme.md

```diff
@@ -0,0 +1,55 @@
+# WebView2_WinUI3_Sample
+This sample shows off using a WebView2 control in a WinUI 3 Windows SDK Packaged application.
+
+This sample also shows how to optionally update the application to ship packaged together with a fixed version of the WebView2 Runtime, instead of using the version of the WebView2 Runtime that's installed on the user's computer.
+
```

Comment:
Add info like top of other samples' readmes: 
This project was created using Visual Studio 2022[|2019].  It uses [programming language list] in the WebView2 context.

motivation for PR comment: I just opened this project in VS2019 but I think I should reopen it in VS2022, need a hint here.